src/apollo-server/index.ts

```diff
@@ -122,6 +123,9 @@ export class OttofellerApolloServerProject extends TypeScriptAppProject {
       },
     })
 
+    // ANCHOR Setup git hooks with Husky
+    addHusky(this, options)
```

Comment:
@quesabe I don’t really like the idea of checking the `hasHusky` option in `addJusky` function. Isn’t it parent caller’s responsibility? What do you think?

Reply:
IMHO it's a matter of preference. I did it similar to other function. However it was done this way because of the `complexity` rule which did not allow too many conditionals within the constructor and I moved most of them inside the functions. For now the rule is gone and we are free to do it either way. For me it does not make much difference, so I'm ready to follow your lead :).

Reply:
Probably I need to review all the existing functions to move all such conditions outside them.